### PR TITLE
Nearby locations

### PR DESCRIPTION
**Pull request by AntonPuko**

### CODE REVIEW

File: src/stores/NearbyLocationsStore.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    }
+  };
+
+  @computed
+  get all(): Array<NearbyLocation> {
+    return this._nearbyLocationsById.values();
+  }
+
+  @computed
+  get allWithFilledTaps(): Array<NearbyLocation> {
```
Comment (AntonPuko) [inline src/stores/NearbyLocationsStore.js]: not sure about the name

File: src/common/avatars/BaseAvatar.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  onPress?: () => void,
+  size: number,
+};
+
+type Props = BaseAvatarProps & { uri: string };
+
+class BaseAvatar extends React.PureComponent<Props> {
+  render() {
+    return (
+      <StyledTouchableOpacity
```
Comment (jlkalberer) [inline src/common/avatars/BaseAvatar.js]: What is this for?

File: src/common/avatars/BeverageAvatar.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import type { BaseAvatarProps } from './BaseAvatar';
+
+import * as React from 'react';
+import CONFIG from '../../config';
+import BaseAvatar from './BaseAvatar';
+
+type Props = {
```
Comment (jlkalberer) [inline src/common/avatars/BeverageAvatar.js]: Doesn't this work?
```
type Props = {
  ...BaseAvatarProps,
  beverageId: string,
};
```

File: src/stores/NearbyLocationsStore.js (hunk 1 above)
Comment (jlkalberer) [reply src/stores/NearbyLocationsStore.js]: Just call it `nearbyLocations`

Comment (AntonPuko) [reply src/stores/NearbyLocationsStore.js]: better `all` for consistency then , I just wanted to point out that it's filtered in some kind.

File: src/common/avatars/BaseAvatar.js (hunk 2 above)
Comment (AntonPuko) [reply src/common/avatars/BaseAvatar.js]: For onPress event and rounded circle the same size as the image, before the image loaded.

File: src/common/avatars/BeverageAvatar.js (hunk 3 above)
Comment (AntonPuko) [reply src/common/avatars/BeverageAvatar.js]: idk why, but no, it doesn't. it treats props.size as undefined. I still can't get a clear difference between & and ... in flow context, sometimes one works better and vice versa.

**Pull request merged**